addons/crm/tests/test_crm_lead_merge.py

```diff
@@ -97,6 +105,68 @@ def test_lead_merge_internals(self):
         self.assertEqual(merge_opportunity.team_id, self.sales_team_convert)
         self.assertEqual(merge_opportunity.stage_id, self.stage_gen_1)
 
+    @users('user_sales_manager')
+    def test_lead_merge_activities(self):
```

Comment:
We could do a test_merge_method_dependencies method (after existing test_merge_method) testing both activities and calendar events in a single test.